.prettierrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Please move prettierrc, .eslintignore files to under ui directory, as these files are scoped to ui code only.

Is .git-blame-ignore-revs scoped to all codes in repo? If no probably this file should get moved too.

Reply:
Thanks for catching - I totally missed moving prettierrc. It's been a while since I worked with a mono repo.

Reply:
As it is related to git, .git-blame-ignore-revs is per git repo basis.
It makes sense to be at the root directory as we're only using one git repository